modules/test/playwright/tests/announcements-web/announcements-web.spec.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {announcementsPagesTest} from '../../fixtures/announcementsPagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+
+export const testEmptyDistributionScope = mergeTests(
+	announcementsPagesTest,
+	loginTest()
+);
+
+testEmptyDistributionScope(
+	'LPD-18804: Do not have a blank option for distribution scope',
+	async ({announcementsPage, page}) => {
+		await announcementsPage.goto();
+		await announcementsPage.goToCreateNewAnnouncement();
+
+		await page.getByLabel('Distribution Scope').selectOption({index: 0});
```

Comment:
This is a bit confusing, but I understand what this want to test (that the first option is General, and not "blank" like it was before)...

Just wondering, have you tried if something like this would work? 

```
        await announcementsPage.goToCreateNewAnnouncement();

	await expect(
		page.getByLabel('Distribution Scope').selectOption('')
	).not.toBeVisible();
```

Could be that it doesn't work, then I have no objections with your approach ;-)

Reply:
The `toBeVisible` only works on a `locator` unfortunately.

The `selectOption('')` will also fail because that option does not exist. I tried finding a way to assert that this call failed in order to test that the option doesn't exist but was unable to. The only options within Playwright I'm seeing are how to assert a value exists or is selected, not validating that one is not available.

Another thought I had was to call `toHaveValue('')` but by default the `General` option is selected so this will fail as well.

If you have any other ideas to try please let me know.

Reply:
Thank you for the detailed explanation. Unfortunately I have no more ideas, so let's move this forward, as the test is valid!